src/tests/ftest/util/server_utils_params.py

```diff
@@ -435,6 +435,7 @@ class EngineYamlParameters(YamlParameters):
         "common": [
             "D_LOG_FILE_APPEND_PID=1",
             "DAOS_POOL_RF=4",
+            "CRT_EVENT_DELAY=1",
```

Comment:
The default is 10 but we are setting 1 for all CI testing. Are there concerns we won't be testing what will happen in production?

Reply:
I.e. do we expect all DAOS administrators to also override the default?

Reply:
@daltonbohning, each exclusion will take `CRT_EVENT_DELAY` seconds longer. So, I think it makes little sense to prolong CI runs by causing more idle waiting, even if this means we are not testing the default.